jekyll/_posts/2019-07-17-version-0202-released.md

```diff
@@ -0,0 +1,240 @@
+---
+title: "Version 0.20.2 released"
+author: The Nim Team
+---
+
+The Nim team is happy to announce version 0.20.2, which is our second
+release candidate for version 1.0.
+
+To read more about version 0.20 and our plans for version 1.0 and beyond,
+check the [previous release notes](https://nim-lang.org/blog/2019/06/06/version-0200-released.html).
+This is mostly a bugfix release of version 0.20.0 (1.0 RC1).
+
+The only new feature is that `toOpenArray`, our slicing operator,
+is now available for JavaScript backend.
+
+Nim now bootstraps with `--styleCheck:error` flag, meaning that the compiler
```

Comment:
Is this not also a new feature? Was ``-styleCheck:error`` in 0.20.0?

Reply:
Yeah, but it was so broken that it was effectively unusable.